citation.bib

```diff
@@ -0,0 +1,7 @@
+@techreport{OpenPBR,
+  author = {Zap Andersson and Paul Edmondson and Julien Guertault and Adrien Herubel and Alan King and Peter Kutz and Andréa Machizaud and Jamie Portsmouth and Frédéric Servant},
+  title = {OpenPBR Surface},
+  institution = {Academy Software Foundation (ASWF)},
+  year = {2024},
```

Comment:
I'd suggest to add `version` and `month` fields, as we may have subsequent versions published the same year.